Mathlib/MeasureTheory/Measure/OuterMeasure.lean

```diff
@@ -1696,19 +1711,20 @@ theorem trim_sup (m₁ m₂ : OuterMeasure α) : (m₁ ⊔ m₂).trim = m₁.tri
 
 /-- `trim` sends the supremum of a countable family of outer measures to the supremum
 of the trimmed measures. -/
-theorem trim_supᵢ {ι} [Countable ι] (μ : ι → OuterMeasure α) : trim (⨆ i, μ i) = ⨆ i, trim (μ i) :=
-  by
-  simp_rw [← @supᵢ_plift_down _ ι]
-  ext1 s
-  haveI : Countable (Option <| PLift ι) := @Option.countable (PLift ι) _
-  obtain ⟨t, hst, ht, hμt⟩ :=
-    exists_measurable_superset_forall_eq_trim
-      (Option.elim' (⨆ i, μ (PLift.down i)) (μ ∘ PLift.down)) s
-  simp only [Option.forall, Option.elim'] at hμt
-  simp only [supᵢ_apply, ← hμt.1, ← hμt.2]
+theorem trim_supᵢ {ι} [Countable ι] (μ : ι → OuterMeasure α) :
+  trim (⨆ i, μ i) = ⨆ i, trim (μ i) := by
+    simp_rw [← @supᵢ_plift_down _ ι]
+    ext1 s
+    haveI : Countable (Option <| PLift ι) := by exact instCountableOption
+    obtain ⟨t, _, _, hμt⟩ :=
+      exists_measurable_superset_forall_eq_trim
+        (Option.elim' (⨆ i, μ (PLift.down i)) (μ ∘ PLift.down)) s
+    simp only [Option.forall, Option.elim'] at hμt
+    simp only [supᵢ_apply, ← hμt.1]
+    exact supᵢ_congr hμt.2
```

Comment:
Indents are off here; the statement should be double-indented and the body single.